packages/dev/core/src/Engines/webgpuEngine.ts

```diff
@@ -810,6 +810,7 @@ export class WebGPUEngine extends Engine {
             supportSRGBBuffers: true,
             supportTransformFeedbacks: false,
             textureMaxLevel: true,
+            texture2DArrayMaxLayerCount: this._webGLVersion > 1 ? 256 : 128,

```

Comment:
not sure what it means here to have a webglversion (for webgpu)

Reply:
You are right @deltakosh, this value should be set to 2048 for webgpu. Updated in the latest commit.